### PR TITLE
Structured Logging migration: modify part logs of controller

### PR DESCRIPTION
**Pull request by JaneLiuL**
What type of PR is this?
/kind feature

What this PR does / why we need it:
Structured Logging migration: modify server_windows part logs of kube-proxy.
xref:
https://github.com/kubernetes/enhancements/tree/master/keps/sig-instrumentation/1602-structured-logging
https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md
Which issue(s) this PR fixes:
#120

### CODE REVIEW

**Comment by JaneLiuL** [comment]
@Huang-Wei  Others I already modify done, except the Line 443 and Line 450.
I try to modify Line 443 and Line 450 as your mention, but the test fail: 
```
pkg/capacityscheduling/capacity_scheduling.go:443:74: klog.ErrorS(err, "Failed to read elasticQuotaSnapshot from cycleState", "ElasticQuotaSnapshotKey", ElasticQuotaSnapshotKey) used as value
pkg/capacityscheduling/capacity_scheduling.go:448:74: klog.ErrorS(err, "Failed to read preFilterState from cycleState", "preFilterState", preFilterStateKey) used as value
make: *** [Makefile:64: build-scheduler] Error 2
```

detail could be check on https://storage.googleapis.com/kubernetes-jenkins/pr-logs/pull/kubernetes-sigs_scheduler-plugins/235/pull-scheduler-plugins-verify-build-master/1432154330324013056/build-log.txt

File: pkg/capacityscheduling/capacity_scheduling.go [hunk 1]
```diff
@@ -441,14 +441,14 @@ func selectVictimsOnNode(
 	elasticQuotaSnapshotState, err := getElasticQuotaSnapshotState(state)
 	if err != nil {
 		msg := fmt.Sprintf("error reading %q from cycleState: %v", ElasticQuotaSnapshotKey, err)
```
[the remark this answers is not in the document]
Comment (Huang-Wei) [reply pkg/capacityscheduling/capacity_scheduling.go:443]: Bump.

**Review by Huang-Wei — COMMENTED** [review]
@JaneLiuL I didn't see the CI failure above. Also it passed all the CI jobs. Could you confirm it still exists?

BTW: except for one unresolved comment, the other changes LGTM.

**Comment by Huang-Wei** [comment]
Oh, just one another comment:

The [guide](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#name-arguments) mentioned to use carmelCase name as the key:

```
Always use lowerCamelCase, for example use containerName and not container name or container_name.
```

So replace "elasticquota" with "elasticQuota" please.

**Comment by JaneLiuL** [comment]
@Huang-Wei  Thanks for the review, please check the test now, it fail if I change the Line 443 as your comment. :)

**Comment by Huang-Wei** [comment]
> @Huang-Wei Thanks for the review, please check the test now, it fail if I change the Line 443 as your comment. :)

The test failed at L443.

**Comment by JaneLiuL** [comment]
@Huang-Wei  If I change the Line443 as the comment that you raise:
```
	if err != nil {
		return nil, 0, framework.NewStatus(framework.Unschedulable, klog.ErrorS(err, "Failed to read elasticQuotaSnapshot from cycleState", "elasticQuotaSnapshotKey", ElasticQuotaSnapshotKey))
	}
```

So I plan to change back to:
```
	if err != nil {
		msg := fmt.Sprintf("error reading %q from cycleState: %v", ElasticQuotaSnapshotKey, err)
		klog.Errorf(msg)
		return nil, 0, framework.NewStatus(framework.Unschedulable, msg)
	}
```
Is it that ok for you?

**Comment by Huang-Wei** [comment]
> Is it that ok for you?

This might be better:

```go
if err != nil {
	msg := "Failed to read elasticQuotaSnapshot from cycleState"
	klog.ErrorS(err, msg, "elasticQuotaSnapshotKey", ElasticQuotaSnapshotKey)
	return nil, 0, framework.NewStatus(framework.Unschedulable, msg)
}
```

**Comment by seanmalloy** [comment]
It looks like all comment were properly addressed.

/lgtm
/assign @Huang-Wei

**Pull request merged**